packages/jest-environment-webdriver/modules/WebDriverEnvironment.js

```diff
@@ -1,22 +1,31 @@
 const NodeEnvironment = require('jest-environment-node');
 const { Builder, By, until } = require('selenium-webdriver');
 
+const { setBuilderBrowserOptions } = require('./ConfigureBuilder');
+
 class WebDriverEnvironment extends NodeEnvironment {
   constructor(config) {
     super(config);
     const options = config.testEnvironmentOptions || {};
     this.browserName = options.browser || 'chrome';
+    this.browserOptions = options.browserOptions || {};
     this.seleniumAddress = options.seleniumAddress || null;
   }
 
   async setup() {
     await super.setup();
     
-    let driver = new Builder();
+    const builder = new Builder();
+
+    setBuilderBrowserOptions(builder, this.browserOptions);
+
     if (this.seleniumAddress) {
-      driver = driver.usingServer(this.seleniumAddress);
+      builder = builder.usingServer(this.seleniumAddress);
```

Comment:
Does this line work since `builder` is defined as `const builder`?

Reply:
Oh lordy, that does not work :)